person/views.py

```diff
@@ -1,3 +1,11 @@
+import json
```

Comment:
Unused import _json_, delete it, please.

Reply:
I forgot to comment that it was planned, since in theory frontend told me that they fixed it. But I delete it and since they import it when they need it.